Dockerfile.ia-archiver

```diff
@@ -0,0 +1,28 @@
+# Use an official Python runtime as a parent image
+FROM python:3.6-slim
+MAINTAINER enviroDGI@gmail.com
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    git gcc g++
+
+# Set the working directory to /app
+WORKDIR /app
+ARG WEB_MONITORING_DB_URL
+ARG WEB_MONITORING_DB_PASSWORD
+ARG WEB_MONITORING_DB_EMAIL
```

Comment:
It doesn’t look like these are used (also, I’m worried about somehow baking in credentials here). Can we remove them?

Reply:
I think those args are needed when running the image, so the application can grab them as environmental variables.

Reply:
I'll double-check.

Reply:
We use those in the launch of the container from the cronjob definition. Although, after reading through the documentation I'm pretty sure that should be "ENV" and not "ARG". 

https://github.com/ibuys/web-monitoring-ops/blob/5f0efef84d0419185e6aabb86f59942b19d15db1/templates/staging/ia-archiver-cronjob.yaml#L15

Reply:
You don’t need to define `ENV` in the dockerfile to use env vars later. For example, we do all the config for the diff server via env vars in Kubernetes, too (https://github.com/edgi-govdata-archiving/web-monitoring-ops/blob/2095044a5f2ba7d65cc5140f7d8fd01705103440/templates/production/diffing-deployment.yaml#L36-L45), but we don’t need to declare them in `Dockerfile` (https://github.com/edgi-govdata-archiving/web-monitoring-processing/blob/master/Dockerfile).

There are actually quite a few more env vars that can be used, which we document in `.env.example`. I think it’s better to leave them out the dockerfile because otherwise they are likely to get out of sync.